lib/rucoa/yard/definitions_loader.rb

```diff
@@ -69,7 +69,7 @@ def map(code_object, path:)
         # @param path [String, nil]
         # @return [Array<Rucoa::Definitions::Base>]
         def load(path: nil, &block)
-          ::YARD::Registry.clear
+          # ::YARD::Registry.clear
```

Comment:
I had to have this commented out because `YARD.parse_string` sometimes need information about the already parsed information to resolve constants, but commenting out this may also cause other bugs...

Reply:
I finally figured out that the reason why this change will break definition paths becuase it will return ALL definitions, not ones gained from given code.